CHANGELOG.md

```diff
@@ -15,12 +15,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `hypothesis` strategies and roundtrip test for kernels and objectives
 
 ### Changed
-- `torch` numeric types are now loaded lazily
 - Reorganized acquisition.py into `acquisition` subpackage
-- `torch` is imported lazily in `surrogates`
 - Acquisition functions are now their own objects
 - `acquisition_function_cls` constructor parameter renamed to `acquisition_function`
+- `torch`, `gpytorch` and `botorch` are lazy-loaded for improved startup time
```

Comment:
(continued): and lazy-loading `sklearn` would also have the benefit that we could really turn in into an optional dependency ...

Reply:
sure but i think thats low prio
curious: how much loading time has sklearn vs torch anyway?

Reply:
Yes, agree, no high prio...

For the numbers: 
* before: loading botorch roughly took 1.2s in the import tree
* after: sklearn takes rougly 0.5s

Feel free to resolve